zerver/tests/test_signup.py

```diff
@@ -560,3 +560,52 @@ def test_digest_unsubscribe(self):
         self.assertEqual(0, len(ScheduledJob.objects.filter(
                 type=ScheduledJob.EMAIL, filter_string__iexact=email)))
 
+class  RealmCreationTest(AuthedTestCase):
+
+    def test_create_realm(self):
+        # type: () -> None
+        username = "user1"
+        password = "test"
+        domain = "test.com"
+        email = "user1@test.com"
+
+        # Make sure the realm does not exist
+        self.assertIsNone(get_realm("test.com"))
+
+        with self.settings(OPEN_REALM_CREATION=True):
+            # Create new realm with the email
+            result = self.client.post('/create_realm/', {'email': email})
+            self.assertEquals(result.status_code, 302)
+            self.assertTrue(result["Location"].endswith(
+                    "/accounts/send_confirm/%s@%s" % (username, domain)))
+            result = self.client.get(result["Location"])
+            self.assertIn("Check your email so we can get started.", result.content)
+
+            # Visit the confirmation link.
+            from django.core.mail import outbox
```

Comment:
@timabbott Actually some other tests in other Classes was also using the outbox. Importing `outbox` global was making the tests fail.